data/styles/clear/include/priorities_2_BG-top.prio.txt

```diff
@@ -20,8 +20,10 @@ landuse-reservoir                                   # area z12- (also has captio
 landuse-salt_pond                                   # area z1-
 natural-water                                       # area z1- (also has caption z10-)
 natural-water-basin                                 # area z1- (also has caption z10-)
+natural-water-ditch                                 # area z1- (also has caption z10-)
 natural-water-lake                                  # area z1- (also has caption z10-)
 natural-water-lock                                  # area z1- (also has caption z10-)
+natural-water-moat                                  # area z1- (also has caption z10-)
```

Comment:
These are relatively small water features, please make their captions appear on z17-.

Otherwise there might be captions visible on e.g. z11, but no area water because it was filtered out due to a small polygon size.

Reply:
fixed